ui/react-components/api/recurringAppointmentsApi.js

```diff
@@ -8,3 +8,11 @@ export const saveRecurringAppointments = async data => {
         console.error(error);
     }
 };
+
+export const getRecurringConflicts = async request => {
```

Comment:
Similarly recurringConflictsFor, what is request?? Appointment Request?

Reply:
changed